### DISCUSSION TITLE
First Game (WIP)!

### DISCUSSION THREAD

**Discussion by SimplyFriday** [excaliburjs/Excalibur#1964] [Show and tell]
Absolutely loving Excalibur so far! I've been a desktop/web developer for a decade, but never made a game before. Always wanted to, so I joined a game jam 😄 

Jam Link: https://itch.io/jam/callum-uptons-game-jam
Source Code (BSL 1.1): https://github.com/josh-greenlaw/callum-jam-01
CI/CD (main) Site: https://josh-greenlaw.github.io/callum-jam-01

I'll definitely be using Excalibur for more things in the future.

**Comment by eonarheim** [comment]
Bravo this is fantastic! Thanks for sharing!
